protoc-gen-pbandk/jvm/build.gradle.kts

```diff
@@ -30,7 +30,11 @@ tasks.withType<JavaCompile> {
 
 val bootJar by tasks.getting(BootJar::class) {
     archiveClassifier.set("jvm8")
-    launchScript()
+    // The Spring Boot launch script prints a warning about running as root, which corrupts the plugin output. This custom script disables the warning.
+    // Script source: https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-tools/spring-boot-loader-tools/src/main/resources/org/springframework/boot/loader/tools/launch.script
```

Comment:
Instead of `main`, please link to the specific commit. That way it's easier to see if the upstream script has been updated and needs to be copied over again.

Reply:
I see, I pushed again, I think I got the right link now? Is it normal practice to link against the blob or the raw version of the file?

Reply:
Thanks! I usually link to the blob, as you did, but linking to the raw version is also fine in my opinion.